sql/core/src/test/java/test/org/apache/spark/sql/JavaColumnExpressionSuite.java

```diff
@@ -81,10 +81,13 @@ public void isInCollectionCheckExceptionMessage() {
     Dataset<Row> df = spark.createDataFrame(rows, schema);
     Exception e = Assert.assertThrows(Exception.class,
       () -> df.filter(df.col("a").isInCollection(Arrays.asList(new Column("b")))));
-    Arrays.asList("cannot resolve",
-      "due to data type mismatch: Arguments must be same type but were")
-        .forEach(s ->
-          Assert.assertTrue(e.getMessage().toLowerCase(Locale.ROOT)
-            .contains(s.toLowerCase(Locale.ROOT))));
+    System.out.println(e.getMessage().toLowerCase(Locale.ROOT));
```

Comment:
Is it needed?

Reply:
Sorry, I left the debugging code by mistake.